packages/fitness/src/PostWorkout.js

```diff
@@ -301,8 +304,8 @@ const postWorkout = (discord, db) => async (stravaId, activityId) => {
    const channel = await discord.channels.fetch(process.env.CHANNEL_STRAVA);
    const message = existing?.messageId
       ? await channel.messages
-           .fetch(existing.messageId)
-           .then((x) => x.edit({ embeds: [embed] }))
+         .fetch(existing.messageId)
```

Comment:
I think these are from vscode's auto whitespace on save.

Reply:
not a problem!